package.json

```diff
@@ -3,11 +3,8 @@
   "version": "1.0.0",
   "description": "Loads environment variables from .env file",
   "main": "lib/main.js",
-  "directories": {
-    "test": "test"
-  },
   "scripts": {
-    "test": "gulp ci"
+    "test": "lab test/* --coverage && standard"
```

Comment:
[> No configuration. Just drop it in. The easiest way to enforce consistent style in your module/project.](https://github.com/feross/standard#one-style-to-rule-them-all)